applications/nametags/nametags.js

```diff
@@ -1,381 +1,154 @@
-            }
-        }
-        avatars = {};
-    }
-
-    //#endregion
-
-    //#region Menu and app item ------------------------------------------------------------------------------------------------
-
-    function setVisible(visible) {
-        if (visible !== isVisible) {
-            isVisible = visible;
-            if (isVisible) {
-                startUpdating();
-            } else {
-                stopUpdating();
-            }
-            if (button) {
-                button.editProperties({ isActive: isVisible });
-            }
-            Settings.setValue(NAMETAGS_VISIBLE_SETTING, isVisible);
-        }
-    }
-
-    function onMenuItemEvent(event) {
-        var visible;
-
-        if (event === MENU_ITEM) {
-            visible = Menu.isOptionChecked(MENU_ITEM);
-            if (visible !== isVisible) {
-                setVisible(visible);
-            }
-        }
-    }
-
-    function onButtonClicked() {
-        Menu.setIsOptionChecked(MENU_ITEM, !isVisible);  // Triggers onMenuItemEvent().
-    }
-
-    //#endregion
-
-    //#region Set-up and tear-down ---------------------------------------------------------------------------------------------
-
-    function setUp() {
-        var visible;
-
-        log("Version " + versionNumber);
-
-        visible = Settings.getValue(NAMETAGS_VISIBLE_SETTING) === true;
-
-        textSizeOverlay = Entities.addEntity({
-            type: "Text",
-            lineHeight: TEXT_SIZE,
-            visible: false
-        }, "local");
-
-        // Work around Entities.textSize() reporting height value ~2 x the proper value.
-        // This problem started circa version 80.
-        Script.setTimeout(function () {
-            // Entities.textSize() doesn't work straight after overlay is created.
-            var charSize = Entities.textSize(textSizeOverlay, "@");
-            isTextSizeBug = charSize.height / charSize.width > 1.5;
-        }, 500);
-
-        Menu.addMenuItem({
-            menuName: MENU_NAME,
-            menuItemName: MENU_ITEM,
-            shortcutKey: MENU_ITEM_SHORTCUT,
-            isCheckable: true,
-            isChecked: visible
-        });
-        Menu.menuItemEvent.connect(onMenuItemEvent);
-
-        Script.setTimeout(function () {
-            // Wait for other scripts to set themselves up on the tablet so as to avoid contention.
-            tablet = Tablet.getTablet("com.highfidelity.interface.tablet.system");
-            if (tablet) {
-                button = tablet.addButton({
-                    icon: APP_ICON_INACTIVE,
-                    activeIcon: APP_ICON_ACTIVE,
-                    text: APP_NAME,
-                    isActive: visible
-                });
-            }
-            if (button) {
-                button.clicked.connect(onButtonClicked);
-            }
-        }, 2500);
-
-        setVisible(visible);  // Starts updating if visible.
+//
+// Copyright 2023 Overte e.V.
+//
+// Written by Armored Dragon
+
```

Comment:
Apache 2.0 license information is missing
Check out the header here:
https://github.com/overte-org/community-apps/blob/master/applications/aaswitcher/app-aaSwitcher.js